scripts/src/javatests/org/oppia/android/scripts/label/AccessibilityLabelCheckTest.kt

```diff
@@ -285,14 +356,111 @@ class AccessibilityLabelCheckTest {
     val tempFileRelativePath = "app/src/main/AndroidManifest.xml"
     val manifestFile = tempFolder.newFile("testfiles/$tempFileRelativePath")
     manifestFile.writeText(testContent)
+    val exemptionFile = File("${tempFolder.root}/$pathToProtoBinary")
+    val builder = AccessibilityLabelExemptions.newBuilder()
+    builder.addExemptedActivity("app/src/main/java/org/oppia/android/app/home/HomeActivity")
+    val exemptions = builder.build()
+    exemptions.writeTo(exemptionFile.outputStream())
 
-    main(retrieveTestFilesDirectoryPath(), tempFileRelativePath)
+    main(
+      retrieveTestFilesDirectoryPath(),
+      "${tempFolder.root}/$pathToProtoBinary",
+      tempFileRelativePath
+    )
 
     assertThat(outContent.toString().trim()).isEqualTo(
       ACCESSIBILITY_LABEL_CHECK_PASSED_OUTPUT_INDICATOR
     )
   }
 
+  @Test
+  fun testAccessibilityLabel_addRedundantExemption_checkShouldFail() {
+    val testContent =
+      """
+      <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="org.oppia.android">
+          <activity
+              android:name=".app.home.HomeActivity"
+              android:label="@string/administrator_controls_title1" />
+      </manifest>
+      """.trimIndent()
+    tempFolder.newFolder("testfiles", "app", "src", "main")
+    val tempFileRelativePath = "app/src/main/AndroidManifest.xml"
+    val manifestFile = tempFolder.newFile("testfiles/$tempFileRelativePath")
+    manifestFile.writeText(testContent)
+    val exemptionFile = File("${tempFolder.root}/$pathToProtoBinary")
+    val builder = AccessibilityLabelExemptions.newBuilder()
+    builder.addExemptedActivity("app/src/main/java/org/oppia/android/app/home/HomeActivity")
+    val exemptions = builder.build()
```

Comment:
Here & elsewhere: consider using the ``apply{}`` pattern to construct the proto in one statement.

Reply:
Done.